.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,96 @@
+name: "🐛 Bug Report"
+description: "Submit a bug report to help us improve"
+title: "🐛 Bug Report: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
+  - type: textarea
+    id: steps-to-reproduce
+    validations:
+      required: true
+    attributes:
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "When I ..."
+  - type: textarea
+    id: code-to-reproduce
+    attributes:
+      label: "📜 Example code for reproducing this bug"
```

Comment:
Lets keep it consistent across other repositories. This info can be included in the reproduction steps.